ocaml/lambda/lambda.ml

```diff
@@ -1443,6 +1443,16 @@ let mod_field ?(read_semantics=Reads_agree) pos =
 let mod_setfield pos =
   Psetfield (pos, Pointer, Root_initialization)
 
+let alloc_mode_of_primitive_description (p : Primitive.description) =
+  if not Config.stack_allocation then
+    if p.prim_alloc then Some alloc_heap else None
+  else
+    match p.prim_native_repr_res with
+    | (Prim_local | Prim_poly), _ -> Some alloc_local
+    | Prim_global, _ -> Some alloc_heap
```

Comment:
I think we should replace this by `if p.prim_alloc then Some alloc_heap else None`

Reply:
Fixed